l1-contracts/test/foundry/l1/unit/concrete/Utils/UtilsFacet.sol

```diff
@@ -166,6 +166,22 @@ contract UtilsFacet is ZKChainBase {
         s.baseTokenGasPriceMultiplierNominator = _nominator;
     }
 
+    function util_setTotalBatchesCommitted(uint256 _totalBatchesCommitted) external {
+        s.totalBatchesCommitted = _totalBatchesCommitted;
+    }
+
+    function util_getTotalBatchesCommitted() external view returns (uint256) {
```

Comment:
this function should be probably deleted as it is not included as one of the facets in the Utils.sol